assets/texts/zh-CN/Changelog/1.1.html

```diff
@@ -0,0 +1,12 @@
+<ul>
```

Comment:
You don't need to copy the changelog entries if you are not going to translate them. :smiley: